website/static/get-started-faqs.json

```
@@ -17,7 +17,8 @@
 {"question": "Will transactions with a higher \"gas price bid\" be confirmed first?","answer": "<p>There is no notion of mempool on Arbitrum, transactions are processed on a first come first served basis by the Sequencer. Thus, the gas price bid parameter does not influence the order in which a transaction is processed.</p>\n\n<p></p>\n\n","key": "will-transactions-with-a-higher-gas-price-bid-be-confirmed-first"},
 {"question": "Where can I find a list of the current validators of the Arbitrum chains?","answer": "<p>Validation on both Arbitrum One and Arbitrum Nova is currently allow-listed to a committee of public entities. You can see the list of validators <strong><a href=\"https://docs.arbitrum.foundation/state-of-progressive-decentralization#allowlisted-validators\">here</a></strong>. Governance currently has the power to change this status.</p>\n\n<p></p>\n\n","key": "where-can-i-find-a-list-of-the-current-validators-of-the-arbitrum-chains"},
 {"question": "Where can I find the current Data Availability Committee members?","answer": "<p>The Arbitrum Nova chain has a 7-party DAC, whose members can be seen <strong><a href=\"https://docs.arbitrum.foundation/state-of-progressive-decentralization#data-availability-committee-members\">here</a></strong>. Governance has the ability to remove or add members to the committee.</p>\n\n<p></p>\n\n","key": "where-can-i-find-the-current-data-availability-committee-members"},
+{"question": "Can I withdraw my funds from Arbitrum back to Ethereum without going through the sequencer? What about funds that are in a contract?","answer": "<p>Yes, it is possible to permissionlessly send a message from Ethereum to be executed on Arbitrum, and bypassing the Sequencer. You can do this by using the DelayedInbox contract and force-including the message after a certain amount of time has passed (currently ~24 hours). You can find more information about this behavior <a href=\"https://docs.arbitrum.io/sequencer#unhappyuncommon-case-sequencer-isnt-doing-its-job\">here</a>.</p>\n\n<p>Keep in mind that you can execute any message in this way, be it a withdrawal of funds back to Ethereum, or a call to a contract.</p>\n\n<p>You can also find an example of force-inclusion in <a href=\"https://github.com/OffchainLabs/arbitrum-tutorials/tree/master/packages/delayedInbox-l2msg\">this tutorial</a>. </p>\n\n<p></p>\n\n","key": "can-i-withdraw-my-funds-from-arbitrum-back-to-ethereum-without-going-through-the-sequencer-what-about-funds-that-are-in-a-contract"},
```

```suggestion
{"question": "Can I withdraw my funds from Arbitrum back to Ethereum without going through the sequencer? What about funds that are in a contract?","answer": "<p>Yes, it is possible to permissionlessly send a message from Ethereum to be executed on Arbitrum, while bypassing the Sequencer. You can do this by using the DelayedInbox contract and force-including the message after a certain amount of time has passed (currently ~24 hours). You can find more information about this behavior <a href=\"https://docs.arbitrum.io/sequencer#unhappyuncommon-case-sequencer-isnt-doing-its-job\">here</a>.</p>\n\n<p>Keep in mind that you can execute any message in this way, be it a withdrawal of funds back to Ethereum, or a call to a contract.</p>\n\n<p>You can also find an example of force-inclusion in <a href=\"https://github.com/OffchainLabs/arbitrum-tutorials/tree/master/packages/delayedInbox-l2msg\">this tutorial</a>. </p>\n\n<p></p>\n\n","key": "can-i-withdraw-my-funds-from-arbitrum-back-to-ethereum-without-going-through-the-sequencer-what-about-funds-that-are-in-a-contract"},
```

non-blocking nit (FAQ CMS entry has been updated)